src/screens/IndexScreen.js

```diff
@@ -602,6 +616,9 @@ export const IndexScreen = ({ navigation, route }) => {
           }}
         </Query>
       )}
+      {query === QUERY_TYPES.POINTS_OF_INTEREST && (
+        <IndexMapSwitch filter={filterType} setFilter={setFilterType} />
+      )}
```

Comment:
Can this be wrapped with view or can the style in IndexMapSwitch be updated? The Zurück Nach Oben button is not accessible because it is behind the filter. 

![Simulator Screenshot - iPhone 14 - 2023-08-21 at 16 09 09](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/ffda3957-dd9d-4b99-a4cd-7b5c1cdb2852)

Reply:
thanks, i will change to not showing a "back to top" in case of POI list + floating button

Reply:
if we remove "back to top" then how will the list look like? The button appears above the list. Do we want it to be like this?

|list with back to top|list without back to top|
|--|--|
![Simulator Screenshot - iPhone 14 Pro Max - 2023-09-14 at 12 11 31](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/7928b2aa-de11-4118-9ce8-8bc88966e069) | ![Simulator Screenshot - iPhone 14 Pro Max - 2023-09-14 at 12 11 38](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/5495a2be-c344-4304-8b20-510a9baefd88)

Reply:
yes, we cannot position the button more to the middle of the screen. it needs to stay at the most possible button, which sure overlapps the last entry. but the content is still visible and pressable.

one thing could be to add some whitespace below the last entry if we do not show a "back to top", so that the last entry is scrollable the way it is fully visible?

Reply:
Just as an idea, I added View instead of BackToTop component in a very simple way and the whole list can be shown smoothly as in the screenshot.
<img width="882" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/17a33ba1-de21-4bd6-891d-adcd523c5f7b">

Reply:
That's a good idea. But we'll need an extra design for that. 🤷‍♂️

Reply:
Thanks for your suggestions. @d4v3d4vE this would be too much changes visually, which would mean an extra task for design + development.

I decided to keep the "back to top" and add a spacer for POIs:

![Simulator Screen Shot - iPhone 14 - 2023-09-14 at 14 32 28](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/0cb34e42-500c-4421-8c14-3d885497c753)

Reply:
This way we can solve the problem in less time. I think it's a good solution. 👍

Reply:
True, this way it also looks nice!